recipes/server_web2.rb

```diff
@@ -50,12 +50,25 @@
 
 # set php time zone
 php_ini = node['platform_family'] == 'rhel' ? '/etc/php.ini' : '/etc/php5/apache2/php.ini'
-ruby_block 'set php timezone' do
-  block do
-    fe = Chef::Util::FileEdit.new(php_ini)
-    fe.search_file_replace_line(/^;date.timezone =.*/, "date.timezone = #{node['time']['timezone']}")
-    fe.write_file
-  end
+execute 'Setup Apache2 PHP5' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php5/apache2/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
+end
+
+execute 'Setup Apache2 PHP5.5' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php/5.5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php/5.5/apache2/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
+end
+
+execute 'Setup Apache2 PHP' do
+  command "sudo sed -i -e 's/;date.timezone =/date.timezone = UTC/g' /etc/php/5.5/apache2/php.ini"
+  action :run
+  only_if {File.exists?('/etc/php.ini')}
+  notifies :reload, 'service[apache2]', :immediately
 end
```

Comment:
How about modifying `php_ini` attribute to cover all scenarios instead of creating multiple `execute` resources?

PHP Timezone should not be enforced to `UTC`, not everyone is following `UTC`. Setting it default to machine timezone is ideal, perhaps we can add another attribute if need to set it to something else.

I prefer using file edit instead of sed, are you facing any issue with it?